src/CodeGenerator/src/Generator/RequestSerializer/QuerySerializer.php

```diff
@@ -68,9 +68,6 @@ public function generateRequestBuilder(StructureShape $shape): array
                 }
                 MEMBER_CODE';
                 $inputElement = '$v';
-            } elseif ($shape instanceof ListShape || $shape instanceof MapShape) {
-                $body = 'MEMBER_CODE';
-                $inputElement = '$this->' . $member->getName();
```

Comment:
Can you tell me why this was removed?

Reply:
Because as List and Map could be nullable now, we need to go through else statement just below. (to check !== null)

Reply:
Ah, Thank you